packages/react-components/react-drawer/library/src/components/DrawerFooter/useDrawerFooterStyles.styles.ts

```diff
@@ -28,14 +28,14 @@ const useStyles = makeResetStyles({
 
 const useDrawerFooterStyles = makeStyles({
   separator: {
-    ':before': {
+    '::before': {
```

Comment:
I don't think there will be any compatibility issues with this change.

[See](https://stackblitz.com/edit/vitejs-vite-kctxx3?file=index.html).